meilisearch/search-bundle/0.2/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+    "bundles": {
+        "MeiliSearch\\Bundle\\MeiliSearchBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "env": {
+        "MEILISEARCH_URL": "http://127.0.0.1:7700",
+        "MEILISEARCH_API_KEY": "",
```

Comment:
Make sure you are using these in your bundle's config too

Reply:
Fixed.